src/storage_broker/normalizers.py

```diff
@@ -20,27 +25,37 @@ class Validation(object):
     request_id = attr.ib(default=str(uuid.uuid4().hex))
     reason = attr.ib(default=None)
     account = attr.ib(default=None)
+    org_id = attr.ib(default=None)
     reporter = attr.ib(default=None)
     system_id = attr.ib(default=None)
     hostname = attr.ib(default=None)
     size = attr.ib(default=None)
 
+
     @classmethod
     def from_json(cls, doc):
         try:
+            # default dictionary in case we don't have an identity
+            ident = {"identity": {"account": None,
+                                  "internal": {"org_id": None}
```

Comment:
Just as a note here!  There is a JIRA up for moving the "org_id" to the top level of the identity header so this may have to be changed whenever that JIRA is done to accommodate the new changes.

JIRA:  https://issues.redhat.com/browse/RHCLOUD-17653